charts/ingress-nginx/templates/controller-poddisruptionbudget.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if or (and .Values.controller.autoscaling.enabled (gt (.Values.controller.autoscaling.minReplicas | int) 1)) (and (not .Values.controller.autoscaling.enabled) (gt (.Values.controller.replicaCount | int) 1)) }}
+{{- if or (and .Values.controller.autoscaling.enabled (gt (.Values.controller.autoscaling.minReplicas | int) 1)) (and (not .Values.controller.autoscaling.enabled) (gt (.Values.controller.replicaCount | int) 1)) (and .Values.controller.keda.enabled (gt (.Values.controller.keda.minReplicas | int) 1))}}
```

Comment:
Need to take a look at this later. I feel like there should be a better way to handle this, especially because we are now missing a "KEDA not enabled" in the other conditions.

Reply:
Sure! 

After doing the first change is also occurred to me that maybe we can re-write the whole if statement in a less complex way, but still not sure how to make it proper.

Reply:
```suggestion
{{- $replicas := .Values.controller.replicaCount }}
{{- if .Values.controller.autoscaling.enabled }}
{{- $replicas = .Values.controller.autoscaling.minReplicas }}
{{- else if .Values.controller.keda.enabled }}
{{- $replicas = .Values.controller.keda.minReplicas }}
{{- end }}
{{- if gt ($replicas | int) 1 }}
```